psst-gui/src/delegate.rs

```diff
@@ -150,7 +152,12 @@ impl AppDelegate<AppState> for Delegate {
     ) -> Option<Event> {
         if self.main_window == Some(window_id) {
             if let Event::WindowSize(size) = event {
-                data.config.window_size = size;
+                // This is a little hacky, but without it, the window will slowly get smaller each time the application is opened.
+                if !self.size_updated {
+                    self.size_updated = true;
+                } else {
+                    data.config.window_size = size;
+                }
```

Comment:
It would be great to keep an issue related to this open so we can figure out a proper solution.